bin/docker-install-tutorials.sh

```diff
@@ -10,6 +10,9 @@ startup_lite
 # wait until galaxy has started
 galaxy-wait -g $galaxy_instance
 
+# run tutorial install as user galaxy
+su - galaxy
```

Comment:
Can we reuse the `$GALAXY_USER` environment variable from https://github.com/bgruening/docker-galaxy-stable/blob/master/galaxy/Dockerfile#L29 ?

Reply:
good call, definitely.